src/manage-apps/using-appimages.md

```diff
@@ -0,0 +1,53 @@
+# AppImages
+
+The AppImage format packages an application in a single, self-contained file. This format allows for a simplified installation process. Appimage.github.io provides links to download hundreds of applications as AppImages. Once downloaded, an AppImage must be given execute permissions in order to launch.
```

Comment:
> Appimage.github.io provides links to download hundreds of applications as AppImages.

The URL should be hyperlinked. (I see it's hyperlinked later; if you don't want to hyperlink it here too, then change the text to something else that's not a URL, like "the AppImage index," although it would still feel like it should be linked, imo.)